model/src/main/java/com/ge/predix/acs/model/Attribute.java

```diff
@@ -114,4 +118,19 @@ public boolean equals(final Object obj) {
     public String toString() {
         return "Attribute [issuer=" + this.issuer + ", name=" + this.name + ", value=" + this.value + "]";
     }
+
+    public int size() {
```

Comment:
Because this class is visible to ACS users, I would prefer not to have this here because it is an internal concern.